packages/scripts/publish.js

```diff
@@ -1,48 +1,88 @@
 #!/usr/bin/env node
 const path = require('path');
 const fs = require('fs');
-const yarn = require('yarn-api');
 const branch = require('git-branch');
-const util = require('util');
+const spawn = require('cross-spawn');
+const dotenv = require('dotenv');
```

Comment:
no .env file is commited, is it normal ?

Reply:
Sure, it contains my personal github and npm keys :-)
I’ve added it to gitignore, btw

Reply:
Could you add a .env.example file with the key that you need inside in order for this to work?
So one can just create a copy and set the parameters.